src/jquery.multiselect.js

```diff
@@ -480,7 +481,13 @@
 
       var item = document.createElement('li');
       item.className = (isDisabled ? 'ui-multiselect-disabled ' : '')
+                        + (this.options.groupColumns ? ' ui-multiselect-columns' : '')
                         + (option.className || '');
+
+      if (this.options.groupColumnsWidth) {
+        item.style = (item.style != '' ? ';':'') + 'width:'+this.options.groupColumnsWidth+'px';
```

Comment:
I think it would be better to do a looser comparison with `item.style = (item.style ? ';' : '') + 'width...` to catch discrepancies across browsers where it might be null, undefined, or the empty string.